keadm/cmd/keadm/app/cmd/util/scriptrun.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2024 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+func RunScript(scriptPath string) error {
```

Comment:
This function is very simple now, I don't think it needs to be abstracted yet. In keadm, the `exec.Command` is used in many places, We can cansider how to abstract them later.